### DISCUSSION TITLE
Magiclysm: Cataclysm behaviors and remnants

### DISCUSSION THREAD

**Discussion by actual-nh** [CleverRaven/Cataclysm-DDA#46057] [Ideas]
Regarding creating [snippets specific to the Magiclysm](https://github.com/CleverRaven/Cataclysm-DDA/projects/42#card-43977154), one question is how the psychosis/whatever affected mage abilities. Did mages try casting but find their ability to control what they did lacking? Did the abilities themselves go "out of control" and start acting on their own (or at the direction of the mage's subconscious/whatever)? Or did it mostly make a difference in motivation to cast various spells? Or, for that matter, did it prevent police/military mages from reacting properly to events? @KorGgenT?

**Comment by KorGgenT** [comment]
i would say it's more accurate to say that the mages that developed the psychosis had absolutely no self-control. The blob may have increased their power in certain situations, and mages that went into a rage caused terrible destruction just because they could or they were angry enough. i'm not sure it would prevent mages to react to normal events, but when an archmage that has reached the pinnacle of power a decade ago and went into seclusion to develop esoteric research suddenly decides to blow something up, you're just picking up the pieces. lore-wise a dozen or so mages lived in the entirety of the united states (just so you have a scale of how rare this type of power and experience is), and maybe 75% of them either blew themselves and their city (plus surrounding cities) up in some way, or went on a rampage. Think of the magical version of nuclear winter in some places, and you have the aftereffects of an incredibly powerful mage casting a spell that would make them hide it away for noone to see.

**Reply by actual-nh** [reply]
Thanks! That makes me think of adding "Thou shalt not suffer a witch to live" - or "witches, repent of your sins before it is too late for you" or whatever - to the apocalyptic religious snippets (church flyers, graffiti, etc). That's on top of likely anti-mage (and pro-mage!) graffiti likely present before the Cataclysm.

> Think of the magical version of nuclear winter in some places, and you have the aftereffects of an incredibly powerful mage casting a spell that would make them hide it away for noone to see.

I'm not sure what you're meaning by "casting a spell that would make them hide it away for noone to see".

**Reply by KorGgenT** [reply]
i mean if a mage were in his right mind, he would see that this spell is way too powerful to let anyone else know about it. so he hides it in his secret journal of notes or whatnot.

also the witch thing is kinda weird and may not fit in the world from the standpoint of magic being very common pre-WWI. i don't want to get religion involved in my mod if at all possible because i just don't want that kind of baggage.

**Comment by actual-nh** [comment]
Understand re religion; no problem.

With regard to non-religious aspects, people are always going to be making conspiracy theories about any group of people doing something others don't understand. This would be accentuated, at least against non-Technomancer mages, by the changes - very few people were likely alive (even before the Magiclysm, that is) who were alive back then (pre-WWI), unless mages themselves or able to hire them (I would imagine biomancers could do something about old age...). Until the situation got bad - and the average person started to become more paranoid thanks to the psychosis encroaching - this would probably be only/mostly underground kooks.

And, well, at least some mages were indeed (involuntarily) involved in this; the stigma would likely spread under stress, just as some parts of government were involved, but "the Man" as an undifferentiated whole will be blamed (at least by some).

I am **_not_**, BTW, saying this should be the predominant emphasis of added snippets! Just that it's what I've thought of so far for graffiti and maybe a flyer or two.

**Comment by actual-nh** [comment]
Another group that might well get blamed, as far as I can tell from the background: Black dragons (and associated lizardmen). Another question is how _**they**_ reacted to various events (before the game-world "present day").

**Comment by actual-nh** [comment]
Or, for a simpler one, a rumor: "Zombies can sense you when you use <class> spells" - or "Zombies can sense you when you use magic, unless it's a <class> spell".

**Comment by actual-nh** [comment]
How about expansions on craters for localized destruction? Examples:
- Lots of burnt, knocked-over things (stormshaper, kelvinist, or more specifically Fire/Storm Elemental attunement gone nuts; should have scorched-variety zombies);
- Lines of scorched terrain (enhanced version of megablast; possibly kelvinist or Fire Elemental/Sun Mage attunement);
- Areas hit by acid (biomancer, dragon breath; puddles or lines; any acid-hit zombies?)
- Areas that seem to have been shredded by flying thorns (druid)
- Lots of metal shards embedded in things (earthshaper; may have rusted/whatever zombies) and/or the earth itself torn up (also earthshaper - or, for larger-scale, Earth Elemental attunement);
- Frozen or formerly-frozen areas (kelvinist, or possibly Tundra Mage attunement; appropriate zombies - from Aftershock, actually...);
- Lots of very confused/dead frogs (magus);
- Things sliced through in a cone of effect (technomancer - Mega Optical Sneeze Beam) or a line (Force Mage attunement);
- Flattened and frozen areas (Glacier Mage attunement);
- Hardened balls of lava in scorched areas (Vulcanist attunement).

Also, what were the psychological effects on black dragons and on lizardfolk?
